### ISSUE TITLE
[BUG] OrcReader benchmark failure

### ISSUE THREAD

**Issue by karthikeyann** [rapidsai/cudf#10158]
**Describe the bug**
ORC_READER_BENCH crashes with 
```
 terminate called after throwing an instance of 'cudf::logic_error'
   what():  cuDF failure at: /home/karthikeyan/3cudf/cpp/benchmarks/io/orc/orc_reader.cpp:145: Benchmark did not read the entire table
```

**Steps/Code to reproduce bug**
`./benchmarks/ORC_READER_BENCH --benchmark_filter=OrcRead/row_selection/0/4/1/3/0`

or
Run all OrcRead/row_selection benchmarks.
`time ./benchmarks/ORC_READER_BENCH --benchmark_filter=row_selection`

**Additional context**
Discovered while working on https://github.com/rapidsai/cudf/pull/10109

**Comment by GregoryKimball** [comment]
Looks like a duplicate of https://github.com/rapidsai/cudf/issues/9961

**Comment by karthikeyann** [comment]
Still relevant. The issue is not fixed. The benchmark arg is disabled in https://github.com/rapidsai/cudf/pull/10194 
https://github.com/rapidsai/cudf/pull/9963#issuecomment-1004832863
https://github.com/rapidsai/cudf/blob/c01a2a41b7d57a1360324270101d9304fbc9515f/cpp/benchmarks/io/parquet/parquet_reader.cpp#L187-L188

**Comment by GregoryKimball** [comment]
Closing in favor of #11675

**Comment by GregoryKimball** [comment]
@karthikeyann Did I make a mistake here? It seems like we should probably re-open this.

**Comment by vyasr** [comment]
Looking back, I think this was indeed closed for the wrong reasons, but it can be closed. This issue is no longer relevant after #11611 and #11543
